mods/parsoid.js

```diff
@@ -784,7 +784,8 @@ PSP.makeTransform = function(from, to) {
 
     return function(restbase, req) {
         var rp = req.params;
-        if (!req.body || !req.body[from]) {
+        if ((!req.body && req.body !== '')
```

Comment:
@d00rman Empty string is allowed as a value for the string parameter. We might want to disallow that, and add support for swaggers custom `allowEmptyValue` parameter, but later, there's already a pending validation PR that needs to be deployed first.

As for `body === undefined` - here it's impossible, as `body[from]` is a required parameter, so body object is also required.